L/llama_cpp/build_tarballs.jl

```diff
@@ -40,16 +40,37 @@ cmake .. \
     -DCMAKE_INSTALL_PREFIX=$prefix \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
     -DCMAKE_BUILD_TYPE=RELEASE \
+    -DBUILD_SHARED_LIBS=ON \
+    -DLLAMA_BUILD_TESTS=OFF \
+    -DLLAMA_BUILD_EXAMPLES=OFF \
+    -DLLAMA_OPENBLAS=OFF \
     -DLLAMA_NATIVE=OFF \
     $EXTRA_CMAKE_ARGS
 make -j${nproc}
 
 # `make install` doesn't work (2023.03.21)
 # make install
+
+# executables
 for prg in main quantize; do
     install -Dvm 755 "./bin/${prg}${exeext}" "${bindir}/${prg}${exeext}"
 done
 
+# libs
+for lib in libllama; do
+    if [[ "${target}" == *-w64-mingw32* ]]; then
+        install -Dvm 644 "./bin/${lib}.${dlext}" "${libdir}/${lib}.${dlext}"
```

Comment:
Libraries should be installed with 755 permissions, otherwise you can't even dlopen them (we should have an audit pass to correct this, but doing the right thing in the first place is better)